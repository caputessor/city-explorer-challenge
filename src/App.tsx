const App = () => {
  return (
    <div role='main'>
      <h1>Home</h1>
    </div>
  );
}

export default App;