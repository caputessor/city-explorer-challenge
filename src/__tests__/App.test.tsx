import { render } from '@testing-library/react';

import App from '../App';

it('should render App component', () => {
  const { container, getByRole } = render(<App />);

  expect(getByRole('main')).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});