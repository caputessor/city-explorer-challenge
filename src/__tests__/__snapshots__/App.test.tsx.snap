// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component 1`] = `
<div>
  <div
    role="main"
  >
    <h1>
      Home
    </h1>
  </div>
</div>
`;
